suites/upgrade/infernalis-x/stress-split-erasure-code-x86_64/9-workload/ec-rados-plugin=jerasure-k=3-m=1.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# k=3 implies a stripe_width of 1376*3 = 4128 which is different from
+# the default value of 4096 It is also not a multiple of 1024*1024 and
+# creates situations where rounding rules during recovery becomes 
+# necessary. 
+#
+tasks:
```

Comment:
randomized correctness test for rados operations on an erasure coded pool using the jerasure plugin with k=3 and m=1